From e65777cf6e6588a8887d449cfb03e293cc269b1d Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Thu, 17 Apr 2025 11:44:32 +0800
Subject: [PATCH 1048/1058] LATX, doc: recommend oma for AOSC OS

AOSC OS uses oma as its default package manager (apt is supplied as a fallback).

See: https://aosc.io/oma
See: https://github.com/AOSC-Dev/oma
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 README.en.rst | 2 +-
 README.rst    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/README.en.rst b/README.en.rst
index 32461cb2cb..a9f0b0b27f 100644
--- a/README.en.rst
+++ b/README.en.rst
@@ -183,7 +183,7 @@ STEP2:
 
 .. code-block:: bash
 
-    apt install -y gcc nettle pcre2 libffi gnutls glib zlib glib-static libgcrypt-static libgpg-error-static libnfs-static pcre-static zlib-static zstd-static openssl-static
+    oma install -y gcc nettle pcre2 libffi gnutls glib zlib glib-static libgcrypt-static libgpg-error-static libnfs-static pcre-static zlib-static zstd-static openssl-static
 
 
 STEP3:
diff --git a/README.rst b/README.rst
index 2e0b884f9e..9466eb73f6 100644
--- a/README.rst
+++ b/README.rst
@@ -166,7 +166,7 @@ STEP2:
 
 .. code-block:: bash
 
-    apt install -y gcc nettle pcre2 libffi gnutls glib zlib glib-static libgcrypt-static libgpg-error-static libnfs-static pcre-static zlib-static zstd-static openssl-static
+    oma install -y gcc nettle pcre2 libffi gnutls glib zlib glib-static libgcrypt-static libgpg-error-static libnfs-static pcre-static zlib-static zstd-static openssl-static
 
 
 STEP3:
-- 
2.49.0

